Library/Homebrew/cask/cmd/info.rb

```diff
@@ -41,11 +41,12 @@ def self.get_info(cask)
         output << artifact_info(cask) + "\n"
         caveats = Installer.caveats(cask)
         output << caveats if caveats
-        output
+        puts output
```

Comment:
This needs to not `puts` as it's used in `Library/Homebrew/extend/os/Mac/missing_formula.rb`